manifests/config.pp

```diff
@@ -34,16 +34,14 @@
         exec { "Ensure ${_opt} kernel option is set for ${scope}":
           command => "/sbin/grubby --update-kernel=${scope} --args=${_opt}",
           path    => ['/bin','/usr/bin'],
-          unless  => "test `/sbin/grubby --info=${scope} | grep -c ^args=` == \
-                     `/sbin/grubby --info=${scope} | grep ^args= | grep -c ${_opt}`",
+          unless  => "/sbin/grubby --info=${scope} | grep ^args= | test -z \"$(grep -wv ${_opt})\"",
         }
       }
       'absent': {
         exec { "Ensure ${_opt} kernel option is absent for ${scope}":
           command => "/sbin/grubby --update-kernel=${scope} --remove-args=${opt}",
           path    => ['/bin','/usr/bin'],
-          unless  => "test `/sbin/grubby --info=${scope} | grep -c ^args=` == \
-                     `/sbin/grubby --info=${scope} | grep ^args= | grep -vwc ${opt}`",
+          unless  => "/sbin/grubby --info=${scope} | grep ^args= | test -z \"$(grep -w ${_opt}\"",
```

Comment:
I think there was a reason to use ${opt} instead of ${_opt} in 'absent'.

Otherwise someone might request "absent foo" and provide value 1, and if args contain "foo=2" it would not ensure it is removed.

Only this returns 1 as expected
```
cat << EOF | grep '^args=' | test -z "$(grep -w rhgb=3)"
args="rhgb=3 bar=asdf"
args="bar=asdf"
EOF
```
But this returns 0, while I would still like to have rhgm removed:
```
cat << EOF | grep '^args=' | test -z "$(grep -w rhgb=3)"
args="rhgb=5 bar=asdf"
args="rhgb bar=asdf"
EOF
```
```suggestion
          unless  => "/sbin/grubby --info=${scope} | grep ^args= | test -z \"$(grep -w ${opt}\"",
```